### PR TITLE
chore: refactor readme

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

This PR refactors a bit the `README.md` of the npm package:

- it adds installation and usage examples
- it moves the documentation paragraph up
- it moves the traits of Biome after the documentation:
	- I removed the part about other tools
	- I added a trait about not needing Node.js to function

## Test Plan

[Proof read](https://github.com/biomejs/biome/blob/refactor/readme/packages/@biomejs/biome/README.md)

### CODE REVIEW

File: packages/@biomejs/biome/README.md [hunk 1]
```diff
@@ -26,30 +26,50 @@
...
-**Biome** is [MIT licensed](https://github.com/biomejs/biome/tree/main/LICENSE) and moderated under the [Contributor Covenant Code of Conduct](https://github.com/biomejs/biome/tree/main/CODE_OF_CONDUCT.md).
+```shell
+# format files
+npx @biomejs/biome format --write ./src
+# lint files
+npx @biomejs/biome lint ./src
+# run format, lint, etc. and apply the safe suggestions
+npx @biomejs/biome check --apply ./src
+# check all files against format, lint, etc. in CI environments
+npx @biomejs/biome ci ./src
```
Comment (nhedger) [inline packages/@biomejs/biome/README.md:47]:
```suggestion
# format files
npx @biomejs/biome format --write ./src

# lint files
npx @biomejs/biome lint ./src

# run format, lint, etc. and apply the safe suggestions
npx @biomejs/biome check --apply ./src

# check all files against format, lint, etc. in CI environments
npx @biomejs/biome ci ./src
```

**Review by Conaclos — APPROVED** [review]

File: packages/@biomejs/biome/README.md (hunk 1 above)
Comment (nhedger) [reply packages/@biomejs/biome/README.md:47]: I suggest giving the code a bit more room to breathe.

**Comment by nhedger** [comment]
What about a link to the playground?

**Comment by ematipico** [comment]
> What about a link to the playground?

I don't know how to present it and where. Any suggestion?

File: packages/@biomejs/biome/README.md [hunk 2]
```diff
@@ -26,30 +26,53 @@
...
+# lint files
+npx @biomejs/biome lint ./src
 
-**Biome** is [MIT licensed](https://github.com/biomejs/biome/tree/main/LICENSE) and moderated under the [Contributor Covenant Code of Conduct](https://github.com/biomejs/biome/tree/main/CODE_OF_CONDUCT.md).
+# run format, lint, etc. and apply the safe suggestions
+npx @biomejs/biome check --apply ./src
+
+# check all files against format, lint, etc. in CI environments
+npx @biomejs/biome ci ./src
+```
```
Comment (nhedger) [inline packages/@biomejs/biome/README.md:52]:
```suggestion

If you want to give Biome a run without installing it, you can also experiment with our [online playground](https://biomejs.dev/playground/).

```

**Pull request merged**